sdk/timeseriesinsights/test-resources.json

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
Not using bicep?

Reply:
Working on Bicep now. I thought I would start the PR as there are tons of files to look at.

Reply:
Bicep support added. 💪